Dockerfile

```diff
@@ -8,4 +8,4 @@ COPY --from=builder /go/src/github.com/openshift/cluster-image-registry-operator
 RUN useradd cluster-image-registry-operator
 USER cluster-image-registry-operator
 COPY deploy/image-references deploy/0* /manifests/
-LABEL io.openshift.release.operator true
+LABEL io.openshift.release.operator=true
```

Comment:
this is not needed and makes us inconsistent with every other dockerfile in every other operator repo.  please undo this change.

Reply:
updated in current push